### PR TITLE
fixes stale session bug

### PR DESCRIPTION
**Pull request by aesnyder**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -271,3 +280,6 @@ DEPENDENCIES
   spring
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
+
+BUNDLED WITH
```
Comment (joelhawksley) [inline Gemfile.lock]: Do we need this and 285?

**Comment by joelhawksley** [comment]
:+1: consider my one comment non-blocking.

File: Gemfile.lock (hunk 1 above)
Comment (aesnyder) [inline Gemfile.lock]: I'm not sure, I thought this is a feature of the new version of bundler and was desired.

File: app/models/app.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class App < ActiveRecord::Base
```
Comment (joelhawksley) [inline app/models/app.rb]: What about calling this `Project`?

Comment (aesnyder) [inline app/models/app.rb]: Happy to do that, but we did all agree on this naming during our whiteboarding session.

**Comment by joelhawksley** [comment]
I'm OK with this naming. @DavidDufresne thoughts?

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -6,7 +6,13 @@ class ApplicationController < ActionController::Base
 
   private
 
+  def ensure_user
+    redirect_to '/auth/github' unless current_user.present?
+  end
+
   def current_user
-    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+    if session[:user_id]
```
Comment (DavidDufresne) [inline app/controllers/application_controller.rb]: I'd probably go with
``````

File: db/migrate/20150902171923_add_features_tickets_and_acceptance_criteria.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+class AddFeaturesTicketsAndAcceptanceCriteria < ActiveRecord::Migration
+  def change
```
Comment (DavidDufresne) [inline db/migrate/20150902171923_add_features_tickets_and_acceptance_criteria.rb]: Where's the features table?

Comment (DavidDufresne) [inline db/migrate/20150902171923_add_features_tickets_and_acceptance_criteria.rb]: Is it too much of a pain to add these one-by-one?

Comment (DavidDufresne) [inline db/migrate/20150902171923_add_features_tickets_and_acceptance_criteria.rb]: Actually forget splitting them. Just double check we have all the classes and tables you think we need.

**Pull request merged**